traits/has_traits.py

```diff
@@ -1358,8 +1358,14 @@ def trait_get(self, *names, **metadata):
 
         Parameters
         ----------
-        names : list of strings
-            A list of trait attribute names whose values are requested.
+        names : optional
+            List of positional arguments. Either trait attribute names as
+            string arguments or a single list or tuple of trait attribute
+            names whose values are requested.
+        metadata : optional
```

Comment:
Change to NumPyDoc style and remove 'optional', which is redundant

```suggestion
        **metadata
```